docs/testplan/dual_tor/dual_tor_active_active_test_plan.md

```diff
@@ -0,0 +1,68 @@
+| ToR A  link down                    | Verify packet flow after the link from ToR A  to the server NiC goes down                        | Take down the link between ToR A  and server NiC                     | ToR A becomes standby and unhealthy link, ToR B stays active                                                                 | T1 switch receives packet from ToR B(active) but not ToR A(standby)                                                                                                                                                |
+| ToR A link drop                     | Verify packet flow if the link from ToR A to the server NiC starts dropping packets, but stay up | Configure the server NiC simulator to drop packets destined to ToR A | ToR A becomes standby and unhealthy link, ToR B stays active                                                                 | T1 switch receives packet from ToR B(active) but not ToR A(standby)                                                                                                                                                |
+| ToR(A) heartbeat loss               | Verify packet flow after ToR A stops sending heartbeats                                          | Stop the link prober submodule on ToR A                              | ToR A becomes standby and unhealthy link, ToR B stays active                                                                 | T1 switch receives packet from ToR B(active) but not ToR A(standby)                                                                                                                                                |
+| ToR(A) BGP shutdown/startup         | Verify packet flow when admin shutdown/startup BGP sessions on ToR A                             | Admin shutdown/startup the BGP sessions on ToR A                     | ToR A becomes standby during BGP down and change back to active after BGP sessions get re-established, ToR B stays active    | During ToR A is standby(BGP sessions down), T1 switch receives packet only from ToR B(active), after ToR A changes back to active(BGP sessions get up), T1 switch could receive packets from either ToR A or ToR B |
+| ToR(A) reboot                       | Verify packet flow during/after ToR A reboot                                                     | Reboot ToR A                                                         | ToR A becomes standby during reboot and change back to active after reboot, ToR B stays active                               | During ToR A is standby, T1 switch receives packet only from ToR B(active), after ToR A changes back to active, T1 switch could receive packets from either ToR A or ToR B                                         |
```

Comment:
What if the reboot is 'warm-reboot' or 'fast-reboot'?

Reply:
`warm-reboot` and `fast-reboot` are not covered here